add-schema.sh

```diff
@@ -145,7 +145,7 @@ then
   line=$(grep -n '</dependencies>' services/pom.xml | cut -d: -f1)
   finalLine=$(($line - 1))
 
-  projectGroupId='org.geonetwork-opensource.schemas'
+  projectGroupId='${project.groupId}'
```

Comment:
I don't think this is correct, `project.groupId` in `services/pom.xml` it is:

https://github.com/geonetwork/core-geonetwork/blob/68e2d427d2dec6881c17c5de2be1598200d8a590/services/pom.xml#L29

And schemas `groupId` was changed to `org.geonetwork-opensource.schemas`. See for example:

https://github.com/metadata101/iso19139.ca.HNAP/blob/bca7c6774ac7caa5c593f8f2901ff5f153796cc2/pom.xml#L5-L9